plugins/ipam/host-local/backend/disk/backend.go

```diff
@@ -56,10 +56,7 @@ func New(network, dataDir string) (*Store, error) {
 }
 
 func (s *Store) Reserve(id string, ip net.IP, rangeID string) (bool, error) {
-	fname := filepath.Join(s.dataDir, ip.String())
-	if runtime.GOOS == "windows" {
-		fname = strings.Replace(fname, ":", "_", -1)
-	}
+	fname := GetEscapedPath(s.dataDir, ip.String())
```

Comment:
Turns out I missed a bunch, also escaping the colon in the dataDir doesnt work so well when the path includes a drive letter - duh!